README.md

```diff
@@ -6,12 +6,13 @@ This is a [Gatsby](https://www.gatsbyjs.com/) project.
 
 ## Build Site Locally
 1. Install [Node](https://nodejs.org/en/download/) and [Yarn](https://classic.yarnpkg.com/en/docs/install)
-2. Install [Manim](https://docs.manim.community/en/stable/installation.html) along with [LaTex](https://www.latex-project.org/)
+2. Install [LaTex](https://www.latex-project.org/)
```

Comment:
> manim is already sepcified in requirements.txt so it need not be installed separately.

Hmm, what if the user didn't install the system deps required for running manim? (for example ffmpeg)

Reply:
I would simply like to keep a link to manim's docs for installing.

Reply:
> Hmm, what if the user didn't install the system deps required for running manim? (for example ffmpeg)

Ah good point, maybe tell them to install those?
How about this?

```suggestion
2. Install [Manim Dependecies](https://docs.manim.community/en/stable/installation.html) (namely Latex and ffmpeg)
```

Reply:
And link to Latex and ffmpeg installation pages?

Reply:
> And link to Latex and ffmpeg installation pages?

It should be linked in the manim docs, so not needed, I guess

Reply:
Ok 👍